ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/SelectConnectionComponent.java

```diff
@@ -10,7 +10,7 @@
 import io.syndesis.qe.pages.SyndesisPageObject;
 import io.syndesis.qe.pages.connections.list.ConnectionsListComponent;
 
-public class ConnectionSelectComponent extends SyndesisPageObject {
+public class SelectConnectionComponent extends SyndesisPageObject {
```

Comment:
I think original was right against conventions we are trying to keep now, root element is syndesis-integrations-connection-select.